palisade/example-model/src/main/java/uk/gov/gchq/palisade/example/config/ServicesConfigurator.java

```diff
@@ -183,10 +185,11 @@ protected UserService createUserServiceForServer() {
     protected ResourceService createResourceServiceForServer() {
         try {
             Configuration conf = createHadoopConfiguration();
-            HadoopResourceService resource = new HadoopResourceService().conf(conf).cacheService(clientServices.createInternalCacheService());
-            resource.addDataService(clientServices.createClientDataServiceConnection());
-            return resource;
-        } catch (IOException e) {
+            HadoopResourceService hadoopResourceService = new HadoopResourceService().conf(conf).cacheService(clientServices.createInternalCacheService());
```

Comment:
This will change once we update how the Egeria-Palisade-service works. Note that the Egeria resource service is not currently in develop branch.